localstack/services/es/cluster.py

```diff
@@ -171,6 +171,10 @@ def _init_directories(self):
         mkdir(dirs.tmp)
         chmod_r(dirs.tmp, 0o777)
 
+        rm_rf(dirs.backup)
```

Comment:
@whummer @alexrashed @thrau I added this cleanup for the backup directory here to stabilize my tests. Not sure why it's working without the backup directory being pre-created in the current docker runs, but this fixes things for me now and all elastic search tests pass. Thoughts?